src/project/ProjectArea.tsx

```diff
@@ -24,68 +23,68 @@ const ProjectArea = ({
   selectedFile,
   onSelectedFileChanged,
 }: ProjectAreaProps) => {
-  const { files, name: projectName } = useProject();
+  let { files, name: projectName } = useProject();
+  //files = Array(20).fill(files[0]);
   return (
-    <HeadedScrollablePanel
-      heading={
-        <Box p={5} pt={3}>
-          <Text fontSize="xs">
-            <FormattedMessage id="project-name" />
-          </Text>
-          <ProjectNameEditable
-            as="h2"
-            fontSize="3xl"
-            fontWeight="semibold"
-            color="grey.800"
-            button="after"
-            justifyContent="space-between"
-            alignItems="flex-start"
-            lineHeight="1.3"
-            pt={0.5}
-          />
-        </Box>
-      }
-    >
-      <Stack spacing={5} pt={2}>
-        <List flexGrow={1} pl={1} pr={1.5}>
-          {files.map((f) => {
-            const selected = selectedFile === f.name;
-            const select = () => {
-              if (isEditableFile(f.name)) {
-                onSelectedFileChanged(f.name);
-              }
-            };
-            return (
-              <ListItem
-                key={f.name}
-                fontWeight={selected ? "semibold" : undefined}
-                _hover={{
-                  bgColor: "blimpTeal.100",
+    <VStack spacing={5} pt={2} flex="1 0 auto" height={0} alignItems="stretch">
```

Comment:
The spacing looks a little generous between the top of the files list and the editable project name in my opinion.

Reply:
Thanks, I'll take a look. I think this it fell out this way from rearranging components.

Reply:
Halved in b46219c64b4c0cf34766bbc04bf537f42c38e434